Description: Fix building against libpng 1.5.
 From the libpng homepage...
 
 The libpng 1.5.x series continues the evolution of the libpng API,
 finally hiding the contents of the venerable and hoary png_struct and
 png_info data structures inside private (i.e., non-installed) header
 files. Instead of direct struct-access, applications should be using
 the various png_get_xxx() and png_set_xxx() accessor functions, which
 have existed for almost as long as libpng itself. (Apps that compiled
 against libpng 1.4 without warnings about deprecated features should
 happily compile against 1.5, too.
Forwarded: pending...
Author: Fedora packagers of NX
Last-Update: 2012-02-06
--- a/nxcomp/Pgn.cpp	2010-03-01 19:18:59.000000000 +0200
+++ b/nxcomp/Pgn.cpp	2011-09-13 16:35:12.000000000 +0300
@@ -414,7 +414,7 @@
 
   png_read_info(pngPtr, infoPtr);
 
-  if (infoPtr -> color_type == PNG_COLOR_TYPE_PALETTE)
+  if (png_get_color_type(pngPtr, infoPtr) == PNG_COLOR_TYPE_PALETTE)
   {
     png_set_expand(pngPtr);
   }
@@ -565,7 +565,7 @@
 
   png_read_info( pngPtr, infoPtr ) ;
 
-  if (infoPtr -> color_type == PNG_COLOR_TYPE_PALETTE)
+  if (png_get_color_type(pngPtr, infoPtr) == PNG_COLOR_TYPE_PALETTE)
   {
     png_set_expand(pngPtr);
   }
@@ -709,7 +709,7 @@
   png_read_info(pngPtr, infoPtr) ;
 
 
-  if (infoPtr -> color_type == PNG_COLOR_TYPE_PALETTE)
+  if (png_get_color_type(pngPtr, infoPtr) == PNG_COLOR_TYPE_PALETTE)
   {
     png_set_expand(pngPtr);
   }
